doc/make.py

```diff
@@ -199,6 +232,23 @@ def zip_html(self):
                      '-q',
                      *fnames)
 
+    def build_docstring(self):
+        """Build single docstring page"""
+        self._create_build_structure()
+
+        args = ('sphinx-build',
+                '-bhtml',
+                '-d{}'.format(os.path.join(BUILD_PATH, 'doctrees')),
+                '-Dexclude_patterns={}'.format(self.exclude_patterns),
+                SOURCE_PATH,
+                os.path.join(BUILD_PATH, 'html'),
+                os.path.join(SOURCE_PATH, 'temp.rst')
+                )
+        # for some reason it does not work with run_os, but it does if I
+        # directly call the joined command
+        # self._run_os(*args)
+        os.system(" ".join(args))
```

Comment:
@datapythonista here I had the problem that if I did `self._run_os()`, that for some reason it did not work as I wanted (the exclude_patterns seemed to be ignored). While when running the exact same (joined) command in the console of with `os.system(..)` does what I want. 
Any idea?

Reply:
I need to take a look, I think Popen and the functions using it like check_call have some problems if there are spaces in the arguments, may be the exclude patterns need to be between quotes. I'll take a look later. But the reason I wrapped the check_call in a method was exactly to be able to move quickly to os.system or something else. check_call is the preferred way, as it's safer, but it's not the first time I see problems with it.

Reply:
Normally there should be no space in the `exclude_patterns` argument.